include/tfrt/host_context/diagnostic.h

```diff
@@ -33,7 +33,8 @@ namespace tfrt {
 class ExecutionContext;
 
 // This is a simple representation of a decoded diagnostic.
-struct DecodedDiagnostic {
+class DecodedDiagnostic {
```

Comment:
I would rather change the forward declarations to struct, because our internal linter will barf at class member naming.

Reply:
sounds good to me!